### PR TITLE
Uses lodash through single modules.

### PR DESCRIPTION
**Pull request by martinheidegger**
Lodash is a quite big library. This PR uses the lodash packages for each method rather than the whole lodash library.

### CODE REVIEW

File: sass-graph.js [hunk 1]
```diff
@@ -63,20 +67,20 @@ Graph.prototype.addFile = function(filepath, parent) {
 
   var i, length = imports.length, loadPaths, resolved;
   for (i = 0; i < length; i++) {
-    loadPaths = _([cwd, this.dir]).concat(this.loadPaths).filter().uniq().value();
```
Comment (xzyfer) [inline sass-graph.js]: The chained syntax is my personal preference.

**Comment by xzyfer** [comment]
I'm not convinced there is real value in this approach. If for no other reason, the user is likely to already have lodash as a dependency of another project, which would prevent the need to download it twice.

**Comment by martinheidegger** [comment]
In my project sass-graph is the only package referring to lodash and lodash is the bulk of the download speed. I would love if you could merge it.

File: sass-graph.js (hunk 1 above)
Comment (martinheidegger) [inline sass-graph.js]: The chained syntax requires that the whole `lodash` package is available. I wouldn't recommend it in regular projects but it I think it is not at all advisable in libraries.

**Comment by xzyfer** [comment]
Sorry I will not be merging this.

>In my project...

Unfortunately what's best for your specific use case is not indicative of what's best for everyone.

> the user is likely to already have lodash as a dependency of another project

Lodash is most depended on npm package. It's likely most projects will have a dependency that also depends on lodash. This benefits the many at a small cost to the few.

***

IMHO the performance argument is a moot point. HTTP is latency bound more so than bandwidth bound. This change increase latency bound operations (more connections for more packages) where as the current behaviour is bandwidth bound (less connections, larger artefacts).

***

On a personal note I am not will to drop the chained syntax.

**Comment by martinheidegger** [comment]
I am sad about your decision. This project is very important (and awesome!). These few lines of code put a significant amount of code on many users computer and slow down the installation process.

**Pull request closed without merging**